x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -295,7 +295,7 @@ class ReportListingUi extends Component<Props, State> {
             return {
               id: job._id,
               type: source.jobtype,
-              object_type: source.payload.type,
+              object_type: source.payload.objectType,
```

Comment:
Hm, I think this part of the diffs need to remain: 7.x uses different field name for this because of the compatibility shim, which is a deliberate diff between master and 7.x. 

The compatibility shim is gone in master but still exists in 7.x. It was removed in part because it caused this field name to change in the payload interface.